ui/app/AppLayouts/Chat/panels/InlineSelectorPanel.qml

```diff
@@ -58,15 +60,30 @@ Item {
                     color: Theme.palette.baseColor1
                 }
                 Item {
+                    id: tagsItemWrapper
                     Layout.fillWidth: true
                     Layout.fillHeight: true
+                    property real initWidth: 0.0
+                    Connections {
+                        target: Global
+                        function onOpenCreateChatView() {
+                            tagsItemWrapper.initWidth = width;
+                        }
+                    }
```

Comment:
This seems wrong. Why generic `InlineSelectorPanel` knows anything about `onOpenCreateChatView` signal?